serializers/src/test/java/at/willhaben/kafka/serializers/ValidatingDeserializerTest.java

```diff
@@ -18,6 +18,7 @@
 
     private static final String INVALID_MESSAGE_BASE64="AgfuH1IK46u2/TW8J8p+dK3YmWpw7fLm5QhGU3Tw+QJNeVRlc3REYXRh";
     private static final String VALID_MESSAGE_BASE64 = "qqqqqgIH7h9SCuOrtvw1vCfKfnSt2JlqcO3y5uUIRlN08PkCTXlUZXN0RGF0YQ==";
+    private static final String INVALID_SIGNATURE_BASE64 = "qqqqqgFH7h9SCuOrtvw1vCfKfnSt2JlqcO3y5uUIRlN08PkCTXlUZXN0RGF0YQ==";
```

Comment:
Changed 7th letter "I" from line above to "H"

Reply:
Thx :-)